apps/web/src/components/RightSideBarActivities.tsx

```diff
@@ -98,7 +98,12 @@ export const RightSideBarActivites: React.FC = () => {
                 return (
                   <div key={item.id} className='flex flex-1 items-center p-1.5'>
                     <div className='flex-initial w-5/6'>
-                      <Checkbox name='careActivities' value={item.id} label={item.name} />
+                      <Checkbox
+                        name='careActivities'
+                        styles='text-bcDarkBlue accent-bcBlueLink'
```

Comment:
The text isn't blue for the checkbox. Refer https://www.figma.com/file/QuGyUBu1JEkNaGTb8ZfC3l/Care-mapping-design?node-id=626%3A10124